README.md

```diff
@@ -24,7 +24,7 @@ grunt.loadNpmTasks('grunt-contrib-qunit');
 ## Qunit task
 _Run this task with the `grunt qunit` command._
 
-You have chosen to write your unit tests using [QUnit][], you have written a
+You have chosen to write your unit tests using [QUnit](http://qunitjs.com/), you have written a
```

Comment:
I assume it was because you didn't set up an `ssh` key on your `Github` account (See [`cloning using ssh` guide](https://help.github.com/articles/which-remote-url-should-i-use/#cloning-with-ssh-urls)).

If you didn't want to generate an `SSH keypair`, instead, you can use [`https` to clone](https://help.github.com/articles/which-remote-url-should-i-use/#cloning-with-https-urls-recommended) your fork:
```bash
git clone https://github.com/alarimer/grunt-contrib-qunit.git
```